src/storage/link.cpp

```diff
@@ -73,14 +95,39 @@ bool Link::isValid() const
 
 bool Link::operator==(const Link &other) const
 {
-    return (m_linkId == other.linkId() && m_url == other.url() && m_thumbPath == other.thumbPath() && m_title == other.title());
+    return (m_linkId == other.linkId()
+            && m_url == other.url()
+            && m_thumbPath == other.thumbPath()
+            && m_title == other.title()
+            && m_date == other.date()
+            && m_sectionName == other.sectionName());
 }
 
 bool Link::operator!=(const Link &other) const
 {
     return !(*this == other);
 }
 
+QDateTime Link::date() const
+{
+    return m_date;
+}
+
+void Link::setDate(const QDateTime &date)
+{
+    m_date = date;
+}
+
+QString Link::sectionName() const
```

Comment:
Date is enough from model section point of view.

Reply:
Unique date will not merge sections elements - Today, Yesterday, Month